src/msw/listctrl.cpp

```diff
@@ -257,6 +257,11 @@ bool wxListCtrl::Create(wxWindow *parent,
     if ( !MSWCreateControl(WC_LISTVIEW, wxEmptyString, pos, size) )
         return false;
 
+    // LISTVIEW generates and endless stream of LVN_ODCACHEHINT event for a
+    // virtual wxListCtrl, so disabled WS_EX_COMPOSITED.
```

Comment:
Tiny nitpick, but you probably meant
```suggestion
    // virtual wxListCtrl, so disable WS_EX_COMPOSITED.
```